server/routes/user.js

```diff
@@ -4,6 +4,9 @@ const verifyAuth = require('../middleware/verifyAuth');
 const User = require('../models/user');
 const University = require('../models/universities');
 const { BadRequest, GeneralError } = require('../utils/errors');
+const courses = require('../models/courses');
+const groups = require('../models/Group');
+const Group = require('../models/Group');
```

Comment:
One of these groups declarations can be removed. Also I don't think we need line 7 because the courses model is required on line 10.